<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="map">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>621</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(255, 255, 255);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonClearResults">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>210</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 85, 127);
border: 2px solid rgb(255, 255, 255);
border-radius: 10px;
font: 81 8pt &quot;Rockwell Extra Bold&quot;;
font-size: 14px;
font-weight: bold;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Сброс  результата</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSearch">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>10</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 85, 127);
border: 2px solid rgb(255, 255, 255);
border-radius: 10px;
font: 81 8pt &quot;Rockwell Extra Bold&quot;;
font-size: 14px;
font-weight: bold;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>ПОИСК</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="fieldSearch">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 85, 127);
border: 2px solid rgb(255, 255, 255);
border-radius: 10px;
font: 81 8pt &quot;Rockwell Extra Bold&quot;;
font-size: 20px;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="plainText">
     <string>Введите запрос</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAdress">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>60</y>
      <width>221</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 85, 127, 200);
border: 2px solid rgb(255, 255, 255);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelMapType">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>260</y>
      <width>221</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 85, 127, 200);
border: 2px solid rgb(255, 255, 255);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="textMapType">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>270</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 8pt &quot;Rockwell Extra Bold&quot;;
font-size: 14px;
font-weight: bold;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Вид карты:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonSatellite">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>320</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 8pt &quot;Rockwell Extra Bold&quot;;
font-size: 14px;
font-weight: bold;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Спутник</string>
    </property>
    <property name="shortcut">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonHybrid">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>340</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 8pt &quot;Rockwell Extra Bold&quot;;
font-size: 14px;
font-weight: bold;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Гибрид</string>
    </property>
    <property name="shortcut">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonScheme">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>300</y>
      <width>82</width>
      <height>16</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 8pt &quot;Rockwell Extra Bold&quot;;
font-size: 14px;
font-weight: bold;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Схема</string>
    </property>
    <property name="shortcut">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="lineHorizontal">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>290</y>
      <width>118</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkboxIndex">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>170</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 8pt &quot;Rockwell Extra Bold&quot;;
font-size: 14px;
font-weight: bold;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Почтовый индекс</string>
    </property>
   </widget>
   <widget class="QLabel" name="textFullAdress">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>70</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 8pt &quot;Rockwell Extra Bold&quot;;
font-size: 14px;
font-weight: bold;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Полный адрес:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="fieldAdressShow">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>90</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 85, 127);
border: 2px solid rgb(255, 255, 255);
border-radius: 10px;
font: 81 8pt &quot;Rockwell Extra Bold&quot;;
font-size: 14px;
font-weight: bold;
color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMapButtons">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>380</y>
      <width>221</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 85, 127, 200);
border: 2px solid rgb(255, 255, 255);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonLeftArrow">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>420</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 85, 127);
border: 2px solid rgb(255, 255, 255);
border-radius: 10px;
font: 81 8pt &quot;Rockwell Extra Bold&quot;;
font-size: 20px;
font-weight: bold;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>◀</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonTopArrow">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>390</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 85, 127);
border: 2px solid rgb(255, 255, 255);
border-radius: 10px;
font: 81 8pt &quot;Rockwell Extra Bold&quot;;
font-size: 14px;
font-weight: bold;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>▲</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonBottomArrow">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>450</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 85, 127);
border: 2px solid rgb(255, 255, 255);
border-radius: 10px;
font: 81 8pt &quot;Rockwell Extra Bold&quot;;
font-size: 14px;
font-weight: bold;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>▼</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonRightArrow">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>420</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 85, 127);
border: 2px solid rgb(255, 255, 255);
border-radius: 10px;
font: 81 8pt &quot;Rockwell Extra Bold&quot;;
font-size: 20px;
font-weight: bold;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>▶</string>
    </property>
   </widget>
   <widget class="Line" name="lineVertical">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>390</y>
      <width>41</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonPlus">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>400</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 85, 127);
border: 2px solid rgb(255, 255, 255);
border-radius: 10px;
font: 81 8pt &quot;Rockwell Extra Bold&quot;;
font-size: 16px;
font-weight: bold;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonMinus">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>440</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 85, 127);
border: 2px solid rgb(255, 255, 255);
border-radius: 10px;
font: 81 8pt &quot;Rockwell Extra Bold&quot;;
font-size: 16px;
font-weight: bold;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="MagnifyingGlass">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>400</y>
      <width>41</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 34px;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>🔍︎</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
